### ISSUE TITLE
Accessibility Audit: Screen readers do not announce effects of zoom buttons 

### ISSUE THREAD

**Issue by brittnylapierre** [ProjectMirador/mirador#3885]
Description: An accessibility audit of Mirador was performed. It was found that there was not enough auditory feedback for users with screen readers detailing the impact of the zoom (ex, zoom level) upon using zoom options. 

Related: [3541](https://github.com/ProjectMirador/mirador/issues/3541),[3110](https://github.com/ProjectMirador/mirador/issues/3110) 

Supporting Comments  
- LIMITED SIGHT – Windows 10/NVDA 2022.4/ Chrome - Buttons are labeled, including zoom in, zoom out, and reset zoom, but the results on the image are not apparent and they can’t tell if they work with NVDA. 
- DEAF-BLIND – iPhone 13 Pro Max/iOS 17.2/Focus 14 Blue 4th gen/VoiceOver/Safari - Attempted to use zoom buttons but did not observe any noticeable changes in the view. 
- LIMITED SIGHT – Windows 10/Windows Narrator/EDGE - Zoom buttons were visible but with unverifiable zoom levels. 
- Blind – JAWS 2024 HOME/EDGE/WINDOWS 10 - Used zoom in/out on page 34 but could not confirm the result 
- DEAF – BLIND – Ipad Pro 9/iPadOS 17.2/Focus 40 Blue 4th gen/VoiceOver/Safari - Attempted to use zoom buttons, but they did not indicate any noticeable changes. 
- LIMITED SIGHT – iPhone 14 Pro/iOS 17.2/VoiceOver/Safari - No indication from VoiceOver about the functionality of zoom buttons. 
- DEAF-BLIND – Windows 11/laptop/JAWS 2020/PacMate 20 braille/Chrome - Unsure of zoom buttons in providing a different view.
